src/platforms/hosted/cmsis_dap.c

```diff
@@ -402,6 +402,8 @@ int cmsis_dap_jtagtap_init(jtag_proc_t *jtag_proc)
 	jtag_proc->jtagtap_tms_seq     = cmsis_dap_jtagtap_tms_seq;
 	jtag_proc->jtagtap_tdi_tdo_seq = cmsis_dap_jtagtap_tdi_tdo_seq;
 	jtag_proc->jtagtap_tdi_seq     = cmsis_dap_jtagtap_tdi_seq;
+	jtag_proc->dev_shift_ir        = dap_shift_ir;
```

Comment:
These changes and the ones in dap.h/c feel like they belong in a separate PR with the jtagtap.c/h changes